src/puppetlabs/ezbake/core.clj

```diff
@@ -847,22 +847,25 @@ Additional uberjar dependencies:
 (defmethod action "build"
   [_ lein-project build-target]
   (action "stage" lein-project build-target)
+  (exec/exec "bundle" "install" "--path" ".bundle/gems" "--binstubs" ".bundle/bin" :dir staging-dir)
```

Comment:
@underscorgan @demophoon updated (will cherry-pick up to master after this is merged).

Reply:
have you run a test build since this change?